src/History.tsx

```diff
@@ -119,6 +119,26 @@ export const History = connect(
     })
 )(HistoryContainer)
 
+const getComputedStyleValues = (el: HTMLElement, stylePropertyNames: string[]) => {
+    const s = window.getComputedStyle(el);
+    const result: { [key: string]: number } = {};
+    stylePropertyNames.forEach(name => result[name] = parseInt(s.getPropertyValue(name)));
+    return result;
+}
+
+const measure = { 
```

Comment:
ok, I was preparing for the possibility of the other combinatorials (outerHeight, innerWidth...)